hedera-mirror-rest/__tests__/specs/transactions/{idOrHash}/no-params.json

```
@@ -0,0 +1,267 @@
+              },
+              {
+                "account": "0.0.98",
+                "amount": 1,
+                "is_approval": false
+              }
+            ],
+            "valid_duration_seconds": "11",
+            "valid_start_timestamp": "1234567890.000000001"
+          },
+          {
+            "charged_tx_fee": 7,
+            "consensus_timestamp": "1234567890.000000003",
+            "entity_id": null,
+            "max_fee": "33",
+            "memo_base64": null,
+            "name": "CRYPTOTRANSFER",
+            "node": "0.0.3",
+            "nonce": 0,
+            "parent_consensus_timestamp": null,
+            "result": "SUCCESS",
+            "scheduled": true,
+            "bytes": "Ynl0ZXM=",
+            "transaction_hash": "lCDEQjAnWTLkkpv1i/mipNt5niRIU7xzY3h0ZHSituUyLqFrI6UF2mOUvZmywjsG",
+            "transaction_id": "0.0.10-1234567890-000000001",
+            "transfers": [
+              {
+                "account": "0.0.9",
+                "amount": 10,
+                "is_approval": false
+              },
+              {
+                "account": "0.0.10",
+                "amount": -11,
+                "is_approval": false
+              },
+              {
+                "account": "0.0.98",
+                "amount": 1,
+                "is_approval": false
+              }
+            ],
+            "valid_duration_seconds": "11",
+            "valid_start_timestamp": "1234567890.000000001"
+          },
+          {
+            "charged_tx_fee": 7,
+            "consensus_timestamp": "1234567890.000000040",
+            "entity_id": null,
+            "max_fee": "33",
+            "memo_base64": null,
+            "name": "CRYPTOTRANSFER",
+            "node": "0.0.4",
+            "nonce": 0,
+            "parent_consensus_timestamp": null,
+            "result": "DUPLICATE_TRANSACTION",
+            "scheduled": false,
+            "bytes": "Ynl0ZXM=",
+            "transaction_hash": "OxfaENC/745n9QsBD59QDfpPjkVVRPEoWwQZHy7KSrG3ScJJNovCMCP7g/hh7R+0",
+            "transaction_id": "0.0.10-1234567890-000000001",
+            "transfers": [
+              {
+                "account": "0.0.9",
+                "amount": 100,
+                "is_approval": false
+              },
+              {
+                "account": "0.0.98",
+                "amount": 1,
+                "is_approval": false
+              }
+            ],
+            "valid_duration_seconds": "11",
+            "valid_start_timestamp": "1234567890.000000001"
+          }
+        ]
+      }
+    },
+    {
+      "urls": [
+        "/api/v1/transactions/rovr8cn6DzCTVuSAV_YEevfN5jA30FCdFt3Dsg4IUVi_3xTRU0XBsYsZm3L-1Kxv",
+        "/api/v1/transactions/ae8bebf1c9fa0f309356e48057f6047af7cde63037d0509d16ddc3b20e085158bfdf14d15345c1b18b199b72fed4ac6f",
+        "/api/v1/transactions/0xae8bebf1c9fa0f309356e48057f6047af7cde63037d0509d16ddc3b20e085158bfdf14d15345c1b18b199b72fed4ac6f"
```

Shouldn't we be supporting search by the prefix also?
```suggestion
        "/api/v1/transactions/0xae8bebf1c9fa0f309356e48057f6047af7cde63037d0509d16ddc3b20e085158bfdf14d15345c1b18b199b72fed4ac6f",
          "/api/v1/transactions/ae8bebf1c9fa0f309356e48057f6047af7cde63037d0509d16ddc3b20e085158",
  "/api/v1/transactions/0xae8bebf1c9fa0f309356e48057f6047af7cde63037d0509d16ddc3b20e085158"
```

---

Seems i mixed up tickets. This would come in another PR.
Ignore